src/routing/pages/probe/GleanDetails.svelte

```diff
@@ -31,6 +31,11 @@
   // taken from glean dictionary
   const isExpired = (item) => {
     if (willNeverExpire(item.expires)) return false;
+    // if expires is a number, ie a version number, then compare it
+    // to the latest release version (only applied to Fx products)
+    if (!Number.isNaN(item.expires) && item.latest_fx_release_version) {
+      return Number(item.latest_fx_release_version) > item.expires;
```

Comment:
> Does item.latest_fx_release_version keep increasing after the metric expires?

`item.latest_fx_release_version` is the latest release version. Since we get this metadata from the Glean Dictionary (and the dictionary [fetches it from the Product Details API](https://github.com/mozilla/glean-dictionary/blob/main/etl/glean_etl.py#L130)) and it is updated daily, it should 99% track the latest release version and change when there's a new release.

> When expires == latest_fx_release_version doesn't that mean the item is expired?

Ohh I forgot that part. I'll change this to `Number(item.latest_fx_release_version) >= item.expires;` instead. Thanks for catching it!

Reply:
np thanks for the explanation!